docs/profilers/tag_prof.md

```diff
@@ -30,15 +30,42 @@ That's how a report looks like:
 
 ## Instructions
 
-TagProf can only be used with RSpec.
+TagProf can be used with both RSpec and Minitest.
```

Comment:
I'm not sure TagProf is a very relevant name for Minitest. I was wondering if I should not create another profiler called `DirectoryProf` and refer to it in TagProf documentation as a Minitest equivalent. What do you think ?

Reply:
Let's use the `TagProf` name for now. We'll probably add support for other tags beyond "type" in the future (maybe, via integration with some third-party plugins, e.g., `minitest-metadat` or similar).